src/container/HashContainer/Base/index.ts

```diff
@@ -1,11 +1,19 @@
 import { Base, Container } from '@/container/ContainerBase/index';
 
+export const enum HashContainerConst {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Maybe you can remove this comments?

Reply:
Changed to scope disable. Do you think it looks better?
```typescript
export const enum HashContainerConst {
  /* eslint-disable no-unused-vars */
  sigma = 0.75,
  treeifyThreshold = 8,
  untreeifyThreshold = 6,
  minTreeifySize = 64,
  maxBucketNum = (1 << 30)
  /* eslint-enable no-unused-vars */
}
```